### PR TITLE
Add get next list handler

### PR DESCRIPTION
**Pull request by itsmontoya**

### CODE REVIEW

File: routes/api/proxy.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ handlers = ["proxy.GetNext"]
 [[route]]
 method = "GET"
 group = "proxy"
-httpPath = "/nextList/:prefix/:filename"
+httpPath = "/nextList/:prefix/:filename/:maxKeys"
```
Comment (aav66) [inline routes/api/proxy.toml:22]: may be it better to be arg?

like /nextList/:prefix/:filename?maxkeys=100

Comment (itsmontoya) [reply routes/api/proxy.toml:22]: I could, but isn't this a required argument?

Comment (itsmontoya) [reply routes/api/proxy.toml:22]: If it's required, I prefer it to be par of the URL param. I feel that query parameters should be for optional values.

Comment (aav66) [reply routes/api/proxy.toml:22]: it is required.
but args looks much more logical here...

Comment (itsmontoya) [reply routes/api/proxy.toml:22]: If optional, going to keep as such. Much cleaner to handle and follows the required/optional principles

**Pull request merged**